speechbrain/utils/distributed.py

```diff
@@ -120,6 +120,20 @@ def ddp_barrier():
         torch.distributed.barrier()
 
 
+def ddp_broadcast(communication_object, src=0):
+    """In DDP mode, this function will broadcast a tensor to all
+    processes.
+    """
```

Comment:
could you please add the standard SB docstring related to the arguments / returns please ? Other than that, I think this is ready to be merged.